ios/FluentUI/SegmentedControl/SegmentedControl.swift

```diff
@@ -15,6 +15,20 @@ open class SegmentedControl: UIView, TokenizedControlInternal {
         static let iPadMinimumWidth: CGFloat = 375
     }
 
+    open override func didUpdateFocus(in context: UIFocusUpdateContext, with coordinator: UIFocusAnimationCoordinator) {
```

Comment:
Does this happen between redraws? Just making sure this isHidden = true -> isHidden = false doesn't result in a flicker or something weird.

Reply:
This might warrant a comment explaining the logic here.

Reply:
There shouldn't be any flickers, but going to hidden when we lose focus will prevent the ring just staying around on the SegmentedControl once focus moves on to another control, rather than just being able to respond to focus moving around within the SegmentedControl.